docs/advanced-solidus/promotions-system.mdx

```diff
@@ -18,7 +18,10 @@ In order to achieve this level of flexibility, the promotions system is composed
 * **Promotion rules** are responsible for checking whether an order is eligible for a promotion.
 * **Promotion actions** are responsible for defining the discount(s) to be applied to eligible
   orders.
-* **Adjustments** are responsible for storing discount information.
+* **Adjustments** are responsible for storing discount information. Promotion adjustments are
+  recalculated every time the order is updated, to check if their eligibility persists when the
+  state of the order changes. You can [customize how this recalculation][how-to-use-a-custom-promotion-adjuster]
+  behaves with the corresponding `Spree::promotion_adjuster_class` preference.
```

Comment:
Not important now, but if we adopt the page to Diátaxis, it would be better not to diverge from the explanation by talking here about the specific preference.

Reply:
Good catch, I'll try to remove that part.